scripts/install

```diff
@@ -128,14 +128,14 @@ ynh_add_config --template="../conf/default.yml" --destination="$final_path/.conf
 #=================================================
 # BUILD APP
 #=================================================
-ynh_script_progression --message="Building app..." --weight=20
+ynh_script_progression --message="Building app... This will take some time." --weight=15
 
 pushd "$final_path"
 	ynh_use_nodejs
 	corepack enable
 	corepack prepare pnpm@latest --activate
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production pnpm i
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production pnpm run build
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production pnpm install
```

Comment:
Env doesn't need to be prod for installing modules, but doesn't matter too much either

Reply:
Ok so I will remove it, no need for useless extra code (and complexity for new contributors)